README.md

```diff
@@ -1137,7 +1128,7 @@ For example:
 
 * `delete_values({'a'=>'A','b'=>'B','c'=>'C','B'=>'D'}, 'B')` returns {'a'=>'A','c'=>'C','B'=>'D'}
 
-Note that since Puppet 4.0.0 the equivalent can be performed with the built-in
+Note that since Puppet 4.0.0, you can perform the equivalent with the built-in
```

Comment:
The same phrase was changed a different way for delete_regexp - use the same? (Both are fine though).